### PR TITLE
Np 25885 nfr funding integration - fetch nfr funding by identifier

### PR DESCRIPTION
**Pull request by ketilaa**
New repo for verified fundings. Added NFR integration. When we add the second (like EU), we should probably move some code to a `commons` module. Leaving it as is for now.

Note: Have some work-in-progress on PagedSearchResult class that I hope we can move to nva-commons eventually. I will make it work properly here first when I add search support in the next PR.

Note 2: Too much in one PR again. Focus on the code if it is a bit overwhelming.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,291 @@
+
```
Comment (tkardu) [inline .gitignore:1]: Should the git ignore file even be here? It shouldnt ?

File: nfr-verified-funding/src/main/java/no/sikt/nva/funding/verified/nfr/client/model/NfrFundingSearchResult.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
+package no.sikt.nva.funding.verified.nfr.client.model;
```
Comment (tkardu) [inline nfr-verified-funding/src/main/java/no/sikt/nva/funding/verified/nfr/client/model/NfrFundingSearchResult.java:1]: SHouldnt that have some page-ination functionality applied too?

File: buildSrc/build.gradle [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+wrapper {
+    distributionType = Wrapper.DistributionType.ALL
+    gradleVersion = "7.2"
+}
+
+dependencies {
+    implementation group: 'nebula.lint', name: 'nebula.lint.gradle.plugin', version: '17.2.3'
+    implementation group: 'org.jacoco', name: 'org.jacoco.core', version: '0.8.7'
+    implementation group: 'org.jacoco', name: 'org.jacoco.report', version: '0.8.7'
```
Comment (fredrikpf) [inline buildSrc/build.gradle:19]: Shouldn't these be taken from the versions.toml-file?

File: nfr-verified-funding/src/test/java/no/sikt/nva/funding/verified/nfr/client/TestClient.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+public class TestClient {
+
+    public static void main(String[] args) throws BadGatewayException {
+        var httpClient = HttpClient.newBuilder().build();
+        var baseUri = URI.create("https://prosjektbanken.forskningsradet.no/prosjektbanken/rest/cristin");
+        var client = new NfrApiClient(httpClient, baseUri);
+
+        var searchResult = client.fetchByProjectId(193391);
+
+        System.out.println(searchResult);
```
Comment (fredrikpf) [inline nfr-verified-funding/src/test/java/no/sikt/nva/funding/verified/nfr/client/TestClient.java:16]: What are you testing/using this method for? As far as I can see you're not using args for anything?

**Review by fredrikpf — APPROVED** [review]

**Pull request merged**